tasks/apply-mapping/tests/test-apply-mapping-tag-expansion.yaml

```diff
@@ -121,29 +121,39 @@ spec:
               set -eux
 
               echo Test that SNAPSHOT contains component comp1
-              test $(cat $(workspaces.config.path)/test_snapshot_spec.json \
-                | jq -r '[ .components[] | select(.name=="comp1") ] | length') -eq 1
+              test "$(
+                jq -r '[ .components[] | select(.name=="comp1") ] | length' \
+                < "$(workspaces.config.path)/test_snapshot_spec.json"
+              )" -eq 1
 
               echo Test that comp1 has the proper tags
-              test $(cat $(workspaces.config.path)/test_snapshot_spec.json \
-                | jq -c '.components[] | select(.name=="comp1") | .tags') == \
-                '["defaultTag","foo-123456","tag1-1980-01-01","tag2-1980-01-01","123456","testrevision-abc","testrev-bar","testrevision","testrev"]'
+              test "$(
+                jq -c '.components[] | select(.name=="comp1") | .tags' \
+                < "$(workspaces.config.path)/test_snapshot_spec.json"
+              )" == '["defaultTag","foo-123456","tag1-1980-01-01","tag2-1980-01-01","123456","testrevision-abc","testrev-bar","testrevision","testrev"]'
```

Comment:
@mmalina, could you suggest the right way to break the line to remove the `line too long` error? I tried a few methods, but they broke the test.

Reply:
You can either split the line using `\`, but be sure to start from the first column on the next line (same level as the `test` command):
```
... '["defaultTag","foo-123456", '\
'"tag1-1980-01-01","tag2-1980-01-01","123456",...'
```

But maybe a better option would be to prepare the expected list separately. And I would suggest to use space-separated string instead of a json array.
```
expectedTags="defaultTag foo-123456"
expectedTags="${expectedTags} tag1-1980-01-01 tag2-1980-01-01"
```

And then the jq would be:
```
jq -cr '.components[] | select(.name=="comp1") | .tags[]'
```